src/pages/index.js

```diff
@@ -26,26 +29,114 @@ const addCardFormValidator = new FormValidator(
   document.querySelector("#add-card-form")
 );
 
+const profileUserInfo = new UserInfo(".profile__title", ".profile__subtitle");
+
+//Create & Render Card
+
+function createCard(data) {
```

Comment:
Needs correction: After submitting the profile modal the submit button text disappears